courageous_comets/__main__.py

```diff
@@ -104,6 +114,8 @@ async def main() -> None:
 
     await init_nltk()
     redis = await init_redis()
+    # Create the search indexes on Redis
+    await create_indexes(redis)
```

Comment:
What happens if the indexes already exist?

Reply:
There is an `overwrite_existing` flag